docs/design/event_compression.md

```diff
@@ -47,7 +47,7 @@ Instead of a single Timestamp, each event object [contains](https://github.com/G
 
 Each binary that generates events:
  * Maintains a historical record of previously generated events:
-   * Implemented with ["Least Recently Used Cache"](https://github.com/golang/groupcache/blob/master/lru/lru.go) in [```pkg/client/record/events_cache.go```](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/pkg/client/record/events_cache.go).
+   * Implemented with ["Least Recently Used Cache"](https://github.com/golang/groupcache/blob/master/lru/lru.go) in [```pkg/client/record/events_cache.go```](../../pkg/client/record/events_cache.go).
```

Comment:
I saw this in another PR. Code links aren't going to work on gh_pages. I think we should punt on that, though. I don't see a good solution.